README.md

```diff
@@ -180,6 +188,30 @@ To build the x64 target (recommended):
 3. `cmake --preset x64`
 4. `cmake --build --preset x64`
 
+To build the ARM target:
+
+1. Open an "x64 Native Tools Command Prompt for VS 2022 Preview".
+2. Change directories to the previously cloned `STL` directory.
+3. `"%VSINSTALLDIR%VC\Auxiliary\Build\vcvarsall.bat" amd64_arm`

```

Comment:
This results in a horrible mess for the INCLUDE, PATH, etc. environment variables. It works, but I think it's easier to depict the default path and tell people to adjust accordingly if they've customized the location.